lib/Elastica/Transport/HttpAsync.php

```diff
@@ -65,7 +65,7 @@ public function exec(Request $request, array $params)
         }
 
         $command_params_string = implode(' ', $command_params);
-        $curl_command = sprintf('nohup %s %s %s > /dev/null', $command, $command_params_string, $baseUri);
+        $curl_command = sprintf('nohup %s %s %s &> /dev/null', $command, $command_params_string, $baseUri);
```

Comment:
Cannot be a space between `&>` and `/dev/null`.

Reply:
ok, going to change it